Overlord.Api/Controllers/MetricsController.cs

```diff
@@ -5,7 +5,6 @@
 using Overlord.Application.CommandHandlers;
 using Overlord.Application.QueryHandlers;
 
-
```

Comment:
I think it is fine do this manually once, but lets start relying on Roslyn Analyzers/[.editorcong](https://editorconfig.org/) to deal with this stuff in the future. I created a task on the board. Same for the other style changes.